DOCS-439: RU review. EN translation. Data storage policies. (#7597)
ClickHouse/ClickHouse: docs/en/query_language/alter.md

#### MOVE PARTITION|PART {#alter_move-partition}

Moves partitions or data parts to another volume or disk for `MergeTree`-engine tables. See [Using Multiple Block Devices for Data Storage](../operations/table_engines/mergetree.md#table_engine-mergetree-multiple-volumes).

```sql
ALTER TABLE table_name MOVE PARTITION|PART partition_expr TO DISK|VOLUME 'disk_name'
```

The `ALTER TABLE t MOVE` query:

- Not replicated, because different replicas can have different storage policies.

> - Returns an error if the configuration contains nonexistent disk or volume. The query also returns an error if the conditions of data moving, that specified in the storage policy, can't be applied.

Reviewer: ```suggestion
- Returns an error if the specified disk or volume is not configured. Query also returns an error if conditions of data moving, that specified in the storage policy, can't be applied.
```

also we could make a separate sub-list with all possible error reasons
